first_order_model/utils.py

```diff
@@ -244,7 +245,7 @@ def get_model_macs(log_dir, generator, kp_detector, device, lr_size, image_size,
             print('{}: {:.4g} G'.format('decoder macs', decoder_macs / 1e9))
             model_file.write('{}: {:.4g} G\n'.format('decoder macs', decoder_macs / 1e9))   
 
-    return decoder_macs
+    return decoder_macs + bottleneck_macs
```

Comment:
Do not overload this function - its name suggests it is getting entire model macs. Call this function from a different function, or pass a dictionary of all macs, and just extract these two fields